perception/euclidean_cluster/src/euclidean_cluster_node.cpp

```diff
@@ -57,10 +57,7 @@ void EuclideanClusterNode::onPointCloud(
   cluster_pub_->publish(output);
 
   // build debug msg
-  if (debug_pub_->get_subscription_count() < 1) {
-    return;
-  }
-  {
+  if (debug_pub_->get_subscription_count() >= 1) {
```

Comment:
```suggestion
  if (1 <= debug_pub_->get_subscription_count()) {
```
it is easier to understand that the left is small.

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/2108/commits/1246d3b592bdd66f8bbe9718454f01356a115062